code/modules/mob/living/basic/space_fauna/netherworld/migo.dm

```diff
@@ -41,6 +41,10 @@
 		new /mob/living/basic/migo/hatsune(get_turf(loc), mapload)
 		return INITIALIZE_HINT_QDEL
 
+	if(istype(src, /mob/living/basic/migo/hatsune))
+		var/static/list/death_loot = list(/obj/item/instrument/piano_synth)
+		AddElement(/datum/element/death_drops, death_loot)
```

Comment:
Unsure how byond's inheritance works, so I added it here since I was afraid that making a new /mob/living/basic/migo/hatsune/Initialize would basically would that remove the addelement/adcomponent below it.